resources/profiles/Peopoly/process/fdm_process_peopoly_common.json

```diff
@@ -84,7 +84,7 @@
     "tree_support_branch_angle": "45",
     "tree_support_wall_count": "0",
     "detect_thin_wall": "0",
-    "top_surface_pattern": "monotonicline",
+    "top_surface_pattern": "monotonic",

```

Comment:
Orca now recommends using the monotonic line pattern for the top surface. Hence, I have updated all profiles to use the monotonic line recently.

I'm not sure if this change was intended or just an unintended result of being overwritten. I just wanted to mention it here